ci-operator/config/syndesisio/syndesis-qe/syndesisio-syndesis-qe-1.15.x__fuse-online-ocp4.13-lp-interop.yaml

```diff
@@ -0,0 +1,67 @@
+base_images:
+  cli:
+    name: "4.13"
+    namespace: ocp
+    tag: cli
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+images:
+- context_dir: docker
+  dockerfile_path: Dockerfile
+  to: fuse-online-test-runner
```

Comment:
I'm not completely sure but I believe this is the image is the env that we will want to run the PQE test ref with.

If that's true then we should put this image into the `from:` in the rey.yaml